dask_image/ndmeasure/_utils/__init__.py

```diff
@@ -1,15 +1,9 @@
 # -*- coding: utf-8 -*-
-
-
-from __future__ import division
-
 import warnings
 
-import numpy
-
 import dask
-import dask.array
-
+import dask.array as da
+import numpy as np
 
 try:
     from dask.array import blockwise as da_blockwise
```

Comment:
@jakirkham is this try statement intended to support a very old version of Dask? Do you think we could get rid of this?

Reply:
Yeah this was renamed in Dask 1.1.0 ( https://github.com/dask/dask/pull/4348 ). If we are comfortable making that a minimum requirement, we can drop that and just use `dask.array.blockwise` below

Reply:
Ok!
I suspect that changing the minimum requirement for Dask is going to involve some fiddling with the environment files we use for the CI, so I'll open an issue & that can be a separate PR.

Reply:
Issue made here: https://github.com/dask/dask-image/issues/232